core/two-steps.ts

```diff
@@ -0,0 +1,45 @@
+import { router } from '@slowreader/core'
+import { atom } from 'nanostores'
+
+interface BackRoutes {
+  [key: string]: string
+}
+
+export type Side = 'first' | 'second'
+
+export const backRoutes: BackRoutes = {
+  categories: '/feeds/categories',
+  fast: '/fast/:category',
+  slow: '/slow/:feed'
+}
+
+export const side = atom<Side>('first')
+
+export function toggleSide(event: KeyboardEvent): void {
+  if (event.key === 'Escape') {
```

Comment:
I am not sure that this code can be used in desktop app or iOS app, since they will not have `KeybordEvent`.

Reply:
If we will create a client for iOS on React Native, isn’t there `KeyboardEvent`? Will the desktop version be on Electron.js?

Reply:
I am planning full native clients. For instance, for GNOME, I am planning to use GTK bindings to JS.

This is why we separate logic between core and clients.

Reply:
Should we implements separate `Escape` button support for each platform?

Reply:
We can listen for Escape button in client code, then listener will run core's function.